src/app/backend/resource/pod/podcommon.go

```diff
@@ -30,14 +32,77 @@ func getRestartCount(pod api.Pod) int32 {
 	return restartCount
 }
 
+// getDisplayPods gets the status of the pod for display in a compact view like a list view. Logic is copied from kubectl ResourcePrinter implementation.
+func getDisplayStatus(pod api.Pod) string {
```

Comment:
Can we have tests for this fuction? Looks like an easy candidate for a few unit tests.